extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -369,23 +389,39 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
   }
 
   /**
-   * Triggers a remote attachment preview URL animation, and redirects
-   * to the specified URL.
+   * Triggers a remote attachment preview URL animation on mobile,
+   * and redirects to the specified URL.
    * @private
    */
   openRemoteV2_() {
-    const clickTarget = this.element.parentElement
-      .querySelector('.i-amphtml-story-page-open-attachment-host')
-      .shadowRoot.querySelector('a.i-amphtml-story-page-open-attachment');
+    // If the element is an amp-story-page-outlink the click target is its anchor element child.
+    // This is for SEO and analytics optimisation.
+    // Otherwise the element is the legacy version, amp-story-page-attachment with an href,
+    // and a click target is the button built by the component.
+    const programaticallyClickOnTarget = () => {
+      const pageOutLinkChild = this.element.parentElement
+        .querySelector('amp-story-page-outlink')
+        ?.querySelector('a');
+
+      const pageAttachmentChild = this.element.parentElement
+        ?.querySelector('.i-amphtml-story-page-open-attachment-host')
+        .shadowRoot.querySelector('a.i-amphtml-story-page-open-attachment');
+
+      if (pageOutLinkChild) {
```

Comment:
Nit: `pageOutlinkChild` (fix case) since `outlink` is used as a single word everywhere